apps/thehive/analyzers/Vast/README.MD

```diff
@@ -0,0 +1,67 @@
+# Vast Neuron
+
+VAST is an embeddable telemetry engine for structured event data, purpose-built for use cases in security operations. VAST is an acronym and stands for Visibility Across Space and Time.
+
+The analyzer takes following input datatype:
+1. ip
+2. subnet (custom)
+3. domain
+4. hash
+
+## How to Debug
+- Create `/input` and `/output` folders in where the analyzer script runs
+- Put a file named `input.json` into input folder with example input:
+    ```
+    {
+        "data": "10.12.14.101",
+        "dataType": "ip",
+        "tlp": 2,
+        "pap": 2,
+        "message": "",
+        "parameters": {},
+        "config": {
+        "proxy_https": null,
+        "cacerts": null,
+        "check_tlp": false,
+        "max_tlp": 2,
+        "auto_extract_artifacts": false,
+        "max_events": 30,
+        "jobCache": 10,
+        "check_pap": false,
+        "max_pap": 2,
+        "endpoint": "127.0.0.1:42000",
+        "jobTimeout": 30,
+        "service": "get",
+        "proxy_http": null
+    }
+    ```
+- While running the script in debug mode, provide `.` as second parameter to indicate `job-directory` is current folder
+- Example `launch.json` file for VSCode
```

Comment:
The instructions (First 3 points) are already IDE agnostic. Last point was provided just an example of this first 3 points integration with VSCode.

Normally debugging instructions are not provided with analyzers but it is really hard to find how to debug those in TheHive documentation. Wanted to add for future references.

Reply:
@dominiklohmann Would my answer suffice your comment? If not, I can simply omit VSCode part and/or add VSCode instructions in a separate file, which does not seem a better alternative:). Let me know how we should resolve this.

Reply:
I think the information is valuable, but it'd make sense to just extract the gist of it rather than providing an editor-specific file that sets everything up for you. What commands do I need to run to achieve the same thing without VSCode?

Reply:
I seperated `how to debug` and `example vscode` sections. `how to debug` section has always been agnostic instructions. All referred files/folders and approach is needed to be able to debug whatever IDE or approach developer is using.

Also for convenience and for future reference, launch.json for VSCode is added there. Let me know if you think there would be any enhancements needed.